docs/maintain/wasp/v1.0.0-rc.6/docs/how-tos/running-an-access-node.md

```diff
@@ -0,0 +1,222 @@
+```
+
+</TabItem>
+<TabItem value="shimmer_testnet" label="ShimmerEVM Testnet">
+
+```
+wget https://files.testnet.shimmer.network/dbs/wasp/latest-wasp_chains_wal.tgz -O - | tar xzv -C data/wasp
+```
+
+</TabItem>
+</Tabs>
+
+:::note Disk Space
+
+Operating as a full archive node require a lot of disk space, we recomment at least 500Gb of free space for now, to operate without issues
+
+:::
+
+## Run Everything
+
+```sh
+docker compose up -d
+```
+
+It will take a few minutes until the hornet node is synced
+
+You can check the sync status by following the logs with `docker logs -f hornet`, or in the web dashboard.
+
+## Download a Wasp-Cli fitting your Wasp Version
+
+You can download and extract Wasp-Cli with:
+
+```
+curl -sL https://github.com/iotaledger/wasp/releases/download/v1.0.3/wasp-cli_1.0.3_Linux_x86_64.tar.gz | tar xzv
+```
+
+## Use Wasp-Cli to obtain peering info
+
+cd into the newly-downloaded wasp-cli directory:
+
+```
+cd wasp-cli_1.0.3_Linux_x86_64/
+```
+
+## Set L1 API Address
+
+```sh
+./wasp-cli set l1.apiaddress {NODE_HOST}
+```
+
+## Set Wasp API Address
+
+```sh
+./wasp-cli wasp add my-node {NODE_HOST}/wasp/api
+```
+
+## Login
+
+```sh
+./wasp-cli login
+```
+```sh Output
+Username: wasp
+Password: (default is wasp)
+Successfully authenticated
+```
+
+## Obtain Peering Info
+```sh
+./wasp-cli peering info
+```
+```sh Output
+PubKey: 0x20a56daa0b5e86b196c37f802089a2b6007a655a12337d287f7313a214af2ec0
+PeeringURL: 0.0.0.0:4000
+```
+Please note the PubKey: 0x20a56daa0b5e86b196c37f802089a2b6007a655a12337d287f7313a214af2ec0 output.
+Send it together with your Ipv4 to node operators that you want to peer with.
+
+## Wait for the other party to peer
+
+… wait …
+
+## Use wasp-cli to add nodes as peers
+
+```sh
+./wasp-cli peering trust peer1 <pubkey> <URL>
+./wasp-cli peering trust peer2 <pubkey> <URL>
+```
+
+## Add Chain
+
+<Tabs groupId="network" queryString>
+<TabItem value="shimmer" label="ShimmerEVM">
+
+```sh
+./wasp-cli chain add shimmer-evm smr1prxvwqvwf7nru5q5xvh5thwg54zsm2y4wfnk6yk56hj3exxkg92mx20wl3s
+```
+
+</TabItem>
+<TabItem value="shimmer_testnet" label="ShimmerEVM Testnet">
+
+```sh
+./wasp-cli chain add shimmer-evm-testnet rms1ppp00k5mmd2m8my8ukkp58nd3rskw6rx8l09aj35984k74uuc5u2cywn3ex
+```
+
+</TabItem>
+</Tabs>
+
+## Activate
+
+
+<Tabs groupId="network" queryString>
+<TabItem value="shimmer" label="ShimmerEVM">
+
+```sh
+./wasp-cli chain activate --chain shimmer-evm
+```
+
+</TabItem>
+<TabItem value="shimmer_testnet" label="ShimmerEVM Testnet">
+
+```sh
+./wasp-cli chain activate --chain shimmer-evm-testnet
+```
+
+</TabItem>
+</Tabs>
+
+## Add Peers as Access Nodes of the Chain
+
+```sh
+./wasp-cli chain access-nodes add --peers=peer1,peer2
+```
+
+## Check if Wasp synced
+
+You can follow the progress by doing docker logs -f wasp. If you chose to create a full-archive node, this will take a long time (several minutes, maybe hours).
```

Comment:
```suggestion
You can follow the progress using `docker logs -f wasp`. If you chose to create a [full-archive node](#run-and-archive-node), this can take several minutes, maybe hours.
```